tern/report/report.py

```diff
@@ -129,11 +129,13 @@ def load_full_image(image_tag_string):
 def generate_report(args, *images):
     '''Generate a report based on the command line options'''
     if args.report_format:
-        return generate_format(images, args.report_format)
-    return generate_format(images, 'default')
+        if args.report_format == 'html':
+            generate_format(images, args.report_format,args)
+        return generate_format(images, args.report_format,args)
+    return generate_format(images, 'default', args)
 
 
-def generate_format(images, format_string):
+def generate_format(images, format_string, args):
```

Comment:
We don't need to add args as an argument here since plugin manager will handle choosing the reporting style.